resources/dcos_user.rb

```diff
@@ -0,0 +1,51 @@
+#
+# Cookbook Name:: dcos
+# Recipe:: default
+#
+# Copyright 2016, Chef Software, Inc.
```

Comment:
Copyright needs to be updated here.

Reply:
Done. The assignment, I assume, is fine.